### ISSUE TITLE
[debug]#30 not fixed

### ISSUE THREAD

**Issue by haptear** [minhhungit/ConsoleTableExt#32]
If multiple rows of data contain UTF8 characters, the output is still misformatted.
I think the final solution should be to override the 'string.format' method to calculate the final number of spaces that should be added in real time.

```
var results = builder.FormattedRows.Select(row => string.Format(tableRowContentFormatNoUtf8Charas, row.ToArray())).ToList();

change to 

 string.FormatEx(tableRowContentFormatNoUtf8Charas, row.ToArray()))
```

![image](https://user-images.githubusercontent.com/16282949/128597591-24be13ea-15e6-469b-852f-7a33bb0d912f.png)

**Comment by haptear** [comment]
test code

```csharp

         var tableData =new List<object>() { 
                new { Id="其123中x213", Name="tab其它语言test", Host="127.0.0.1", Port=80, status ="success" } ,
                new { Id="中中123午", Name="中123午tes好demo", Host="127.0.0.1", Port=80, status ="success" } ,
            };
            
            ConsoleTableExt.ConsoleTableBuilder
                .From<dynamic>(tableData)
                //.WithTitle("ALL CENTERS", ConsoleColor.Yellow, ConsoleColor.DarkGray)
                .WithFormat(ConsoleTableBuilderFormat.Alternative)
                .ExportAndWriteLine(TableAligntment.Left);

            Console.ReadLine();
```

**Comment by haptear** [comment]
@minhhungit  please review RP

**Comment by minhhungit** [comment]
thanks @haptear for the PR, merged and released in version 3.1.9
